releasenotes/notes/1.3/deprecate-pulse-package-07a621be1db7fa30.yaml

```diff
@@ -34,7 +34,6 @@ deprecations_qpy:
 deprecations_transpiler:
   - |
     As part of the Qiskit Pulse package deprecation, pulse-related aspects in the :class:`qiskit.transpiler.Target` class are being deprecated. These include:
-      * :attr:`~qiskit.transpiler.Target.calibration`
```

Comment:
Why removing this?

Reply:
There's no attribute (or method) called `calibration` 🙂

Reply:
It's `InstructionProperties.calibration`: https://docs.quantum.ibm.com/api/qiskit/qiskit.transpiler.InstructionProperties#calibration probably.

Reply:
Oh, yes, it's in `InstructionProperties` and covered already 😄